databases/src/create/PostgresAccountWizard/createPostgresServer/PostgresServerCreateStep.ts

```diff
@@ -67,8 +68,8 @@ export class PostgresServerCreateStep extends AzureWizardExecuteStep<IPostgresSe
             version: "12",
             administratorLogin: parameters.administratorLogin,
             administratorLoginPassword: parameters.administratorLoginPassword,
-            storageProfile: {
-                storageMB: parameters.storageMB
+            storage: {
```

Comment:
This is the only place where I saw GB come in. 🤷‍♂️

Reply:
I changed the SKUs themselves because I thought that was safer. You can copy them if you want.
https://github.com/microsoft/vscode-cosmosdb/blob/main/src/postgres/commands/createPostgresServer/steps/PostgresServerSkuStep.ts